docker/thehive/test

```diff
@@ -28,10 +28,10 @@ docker compose run --no-TTY vast import --blocking suricata \
 
 # add the app to the stack
 COMPOSE_FILE="$COMPOSE_FILE:compose/thehive.app.yaml"
-docker compose up --build -d thehive-app
+docker compose up --build --no-recreate -d thehive-app
 
 # check that the historical import took place
-docker compose run thehive-app count-alerts 2>&1 | \
+docker compose run --no-deps thehive-app count-alerts 2>&1 | \
```

Comment:
The key here are these `--no-recreate` and `--no-deps` which avoid that TheHive init gets interrupted in the middle, leaving it in an undefined state